p2pd/main.go

```diff
@@ -106,6 +106,8 @@ func main() {
 	idleTimeout := flag.Duration("idleTimeout", 0,
 		"Kills the daemon if no client opens a persistent connection in idleTimeout seconds."+
 			" The zero value (default) disables this feature")
+	persistentConnMaxMsgSize := flag.Int("persistentConnMaxMsgSize", 1<<22,
```

Comment:
```suggestion
	persistentConnMaxMsgSize := flag.Int("persistentConnMaxMsgSize", 4 * 1024 * 1024,
```

This makes it more clear that the constant is 4 MiB.